trlc/lexer.py

```diff
@@ -108,6 +108,20 @@ def context_lines(self):
         return [stripped_line,
                 " " * stripped_offset + "^" * min(tlen, maxtrail)]
 
+    def get_end_location(self):
+        """Get location pointing to the end of a source."""
```

Comment:
This docstring should go into the initial declaration of the function, in Location